src/components/domain/project/projectLinkModal/ProjectLinkModal.tsx

```diff
@@ -224,3 +300,24 @@ const LinkCompleteMessage = () => {
     />
   );
 };
+
+// 팀원 데이터 정렬 및 비회원/회원 구분선 index 구하기
+const processingMemberData = (memberData: MemberData[]) => {
+  // 회원가입이 안된 팀원 먼저 보여주기 - userId, 이름순 정렬
+  const sortedMemberData = memberData.sort((a, b) => {
+    if (a.userId !== b.userId) {
+      return a.userId !== '-1' ? 1 : -1;
+    }
+    return a.name.localeCompare(b.name);
+  });
+
+  // Separator를 넣어줄 비회원->회원 경계 index값 찾기
+  let separatorIndex = -1;
```

Comment:
저희 컨벤션에서 let 사용은 지양하기로 했던 것 같습니다!
```suggestion
  const separatorIndex = sortedMemberData.findIndex(
  (member, index, arr) => index < arr.length - 1 && member.userId !== arr[index + 1].userId
);
```

Reply:
✅ 8efb1c8eae8b12e10aeb992ca45d5353b7913a94